web/src/hotkey.ts

```diff
@@ -327,7 +340,7 @@ export function get_keydown_hotkey(e) {
     let key = e.key;
     if (!use_event_key) {
         const code = `${e.shiftKey ? "Shift+" : ""}${e.code}`;
-        key = CODE_TO_QWERTY_CHAR[code];
+        key = CODE_TO_QWERTY_CHAR[code]!;
```

Comment:
I'm not sure this type assertion is correct. I think if it's a key code that we don't handle, we should just `return undefined`?

@prakhar1144 can you confirm?

Reply:
Yeah, that's right.